tests/unit/restapi/v1/conftest.py

```diff
@@ -90,3 +90,29 @@ def registered_queues(
         "queue2": queue2_response,
         "queue3": queue3_response,
     }
+
+
+@pytest.fixture
+def registered_tags(
+    client: FlaskClient, db: SQLAlchemy, auth_account: dict[str, Any]
+) -> dict[str, Any]:
+    tag1_response = actions.register_tag(
+        client,
+        name="ML: Object Recognition",
+        group_id=auth_account["default_group_id"],
+    )
```

Comment:
fixtures should return `dict` not `TestResponse`. Add `.get_json()` to the end of each call to `register_tag`

Reply:
Added in commit: [7ace257](https://github.com/usnistgov/dioptra/pull/455/commits/7ace257fc286888bd374327230b9be74fa78ef47)